src/loaders/ult_load.c

```diff
@@ -151,18 +148,13 @@ static int ult_load(struct module_data *m, HIO_HANDLE *f, const int start)
 	uih.sizeend = hio_read32l(f);
 	uih.volume = hio_read8(f);
 	uih.bidiloop = hio_read8(f);
+	uih.c2spd = (ver >= 4) ? hio_read16l(f) : 0; /* Incorrect in ult_form.txt */
```

Comment:
Where is c2spd handled for `ver < 4` cases?

Reply:
The Ultra Tracker format does not store c2spd prior to V004, so it's always 8363Hz. This variable is only used with `libxmp_c2spd_to_note` later in the loader to set instrument transpose and finetune when `ver > 3` (not added by me, I prefer `>=` to  `>` for version checks). The default for those is presumably 0, which should be equivalent to 8363Hz.

Reply:
OK.